### ISSUE TITLE
UsageError: Cell magic `%%send_to_spark` not found.

### ISSUE THREAD

**Issue by nickorka** [jupyter-incubator/sparkmagic#587]
I'm getting this error on version 0.13.1:

![jupyter_sparkmagic](https://user-images.githubusercontent.com/5385562/68804610-1fcdcf80-0630-11ea-8945-658afe7b7119.png)

What is wrong?

**Comment by nickorka** [comment]
@spaszek I'know that you was the main contributor to this functionality. Can you take look at this? 
Let me know if you need more info.

CC: @aggFTW

**Comment by spaszek** [comment]
Well, it has been merged to master branch but no release has been performed. It will be available only with the new version `0.13.2` or `0.14.0`(none of them exist yet). I am not sure who can do the release though. @itamarst do you have correct permissions? 

For now, the only way is to use the new functionality is to install `sparkmagic` directly from git repository on master branch instead of fixed PyPi version. 

```
pip install git+https://github.com/jupyter-incubator/sparkmagic
```

should do the trick I think.

**Comment by itamarst** [comment]
I've just released 0.14.0. It should show up PyPI in a few minutes.
